pkg/rehearse/jobs_test.go

```diff
@@ -431,14 +481,14 @@ func TestExecuteJobsErrors(t *testing.T) {
 	}{{
 		description: "fail to Create a prowjob",
 		jobs: map[string][]prowconfig.Presubmit{targetOrgRepo: {
-			*makeTestingPresubmit("job1", "ci/prow/job1", "master"),
+			*makeTestingPresubmit("job1", "ci/prow/job1", "master", ""),
```

Comment:
Passing empty strings to a function and checking if they are not empty inside is smelly -- can we make a `makeTestPresubmitWithVariant` or similar?

Reply:
Looks like none of the tests needed the variant flags anyway? I'll just revert it back to how it was before. We may need to add some tests for variants in a followup PR though.